services/doctor/storage.go

```diff
@@ -121,7 +121,8 @@ func checkStorage(opts *checkStorageOptions) func(ctx context.Context, logger lo
 					storer: storage.LFS,
 					isOrphaned: func(path string, obj storage.Object, stat fs.FileInfo) (bool, error) {
 						// The oid of an LFS stored object is the name but with all the path.Separators removed
-						oid := strings.ReplaceAll(path, "/", "")
+						oidUnix := strings.ReplaceAll(path, "/", "")
+						oid := strings.ReplaceAll(oidUnix, "\\", "") // strip windows separator
```

Comment:
How about `strings.Join(filepath.SplitList(path), "")`

Reply:
if I understand that function correctly, its splits on the OS $PATH seperator (ie : for unix) not of the path delimiter (ie /)

Reply:
an alternative is to use os.PathSeparator constant as this is OS specific and thus this will be a built-time constant